pkg/gateway/path/resolver_test.go

```diff
@@ -74,6 +74,36 @@ func TestResolvePath(t *testing.T) {
 			args:    args{encodedPath: "-invalid"},
 			wantErr: true,
 		},
+		{
```

Comment:
FTR, the _real_ tests are in pkg/graveler/ref/rev_parse_test.go.  I see no reason to try to replicate them or the regexp here.